client/cli/src/params/keystore_params.rs

```diff
@@ -56,24 +56,37 @@ pub struct KeystoreParams {
 	pub password_filename: Option<PathBuf>,
 }
 
+impl Drop for KeystoreParams {
+	fn drop(&mut self) {
+		use sp_core::crypto::Zeroize;
+		self.password.zeroize();
```

Comment:
Can we not replace `password: Option<StringSecret>` to remove this custom implementation of `Drop` here?

Reply:
It implements FromStr, but structopt complain about to_string not being implemented. Could use a type wrapper, but I am not sure it is worth it.

Reply:
ok, it is actually the from str error that needed to be display, so no pb.